src/field_theory/intermediate_field.lean

```diff
@@ -286,4 +286,29 @@ instance has_lift2 {F : intermediate_field K L} :
 
 end tower
 
+section finite_dimensional
+
+instance finite_dimensional_left [finite_dimensional K L] (F : intermediate_field K L) :
+  finite_dimensional K F :=
+  finite_dimensional.finite_dimensional_submodule F.to_subalgebra.to_submodule
+
+instance finite_dimensional_right [finite_dimensional K L] (F : intermediate_field K L) :
+  finite_dimensional F L :=
+finite_dimensional.right K F L
+
+lemma eq_of_le_of_findim_eq [finite_dimensional K L] {F E : intermediate_field K L} (h_le : F ≤ E)
+  (h_findim : finite_dimensional.findim K F = finite_dimensional.findim K E) : F = E :=
```

Comment:
You can also weaken this to `findim K E \le findim K F`. Might be useful to add that version.